From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 14 Apr 2018 19:10:13 +0200
Subject: [PATCH] {BUG in 4.17-rc1} phy: do not print a uninitialized variable

clang reports:

    drivers/phy/st/phy-stm32-usbphyc.c:364:8: note: initialize the
    variable 'i' to silence this warning
                    int i;
                         ^
                          = 0
---
 drivers/phy/st/phy-stm32-usbphyc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/phy/st/phy-stm32-usbphyc.c b/drivers/phy/st/phy-stm32-usbphyc.c
index bc4e78a19c91..e263e541b308 100644
--- a/drivers/phy/st/phy-stm32-usbphyc.c
+++ b/drivers/phy/st/phy-stm32-usbphyc.c
@@ -368,7 +368,7 @@ static int stm32_usbphyc_probe(struct platform_device *pdev)
 			ret = PTR_ERR(phy);
 			if (ret != -EPROBE_DEFER)
 				dev_err(dev,
-					"failed to create phy%d: %d\n", i, ret);
+					"failed to create phy: %d\n", ret);
 			goto put_child;
 		}
 
-- 
